src/components/organisms/AssetActions/Edit/index.tsx

```diff
@@ -79,6 +96,8 @@ export default function Edit({
         links: typeof values.links !== 'string' ? values.links : []
       })
 
+      ddo.price.type === 'pool' || (await updateFixedPrice(values.price))
```

Comment:
This breaks actual editing, what was the intention of this line?

Was the intention to only fire method when FRE asset? Then more like so maybe, fire when FRE & only when price has been changed:

```js
ddo.price.type === 'exchange' && values.price !== ddo.price.value && await updateFixedPrice(values.price)
```